Update package metadata

simnalamburt opened the pull request:
1.  restbase-mod-table-mysql is not for testing purpose
2.  Updated repository URL and homepage URL
3.  This repository will be jointly maintained by both Wikimedia and Femiwiki
4.  The bugs of this package should be reported to restbase-mod-table-mysql not restbase phabricator.

lens0021 commented on package.json:
```diff
@@ -15,12 +15,15 @@
   "license": "Apache-2.0",
   "bugs": {
-    "url": "https://phabricator.wikimedia.org/tag/restbase/"
+    "url": "https://github.com/wikimedia/restbase-mod-table-mysql/issues"
```
A discussion for where is more suitable for this project is needed. Using Phabricator gives developers and volunteers interested in MediaWiki more opportunities to find issues.

d00rman replied:
Phabricator is the standard place for issues and bug for all Wikimedia-owned repositories, so it'll have to stay that way.

lens0021 replied:
@simnalamburt: Can I fix this myself If you do not have enough time?

simnalamburt replied:
OK it's all yours

lens0021 reviewed (changes requested):
Nice catches.

lens0021 commented:
I have created [a request for creating a new project 'RESTBase-MySQL'](https://phabricator.wikimedia.org/T238631)

simnalamburt commented:
Then I'll close the [issue tracker of this repository](https://github.com/wikimedia/restbase-mod-table-mysql/issues) once "Restbase-MySQL" project in Phabricator is created.

lens0021 commented:
https://phabricator.wikimedia.org/tag/restbase-mysql/ has been created

d00rman commented:
@simnalamburt @lens0021 I noticed that you create branches in this repo and submit them for PRs. To keep things clean and separate, please fork it under your usernames and then create branches there and submit PRs against the main fork. That way the main repo contains only meaningful branches (and is the praxis we have been using for all other repos).

simnalamburt commented:
@d00rman OK I got it!

lens0021 committed:
Update package metadata

1.  restbase-mod-table-mysql is not for testing purpose
2.  Updated repository URL and homepage URL
3.  This repository will be jointly maintained by both Wikimedia and
    Femiwiki
4.  The bugs of this package should be reported to
    restbase-mod-table-mysql not restbase phabricator.
